samples/bluetooth/fast_pair/locator_tag/sysbuild/CMakeLists.txt

```diff
@@ -6,6 +6,9 @@
 
 set(APPLICATION_CONFIG_DIR "${CMAKE_CURRENT_LIST_DIR}/../configuration")
```

Comment:
Shouldn't this be done in application's `CMakeListst.txt`? (instead of sysbuild's `CMakeListst.txt`)

Reply:
Comment no longer applies after PR was updated. Resolving.